neqo-http3/src/connection_server.rs

```diff
@@ -136,7 +136,7 @@ impl Http3ServerHandler {
                     ),
                     StreamType::UniDi => self
                         .base_handler
-                        .handle_new_unidi_stream(stream_id.as_u64()),
+                        .handle_new_unidi_stream(stream_id.as_u64(), false),
```

Comment:
Would it be better to pass `Role` here instead of a boolean?  The effect in terms of code is the same, but the code might be more readable as a result.